src/Generator.h

```diff
@@ -3478,6 +3478,9 @@ class GeneratorBase : public NamesInterface, public AbstractGenerator {
 
     const size_t size;
 
+    // Flag to enable serialization (and bypass asserts for unbound target)
+    bool serializable = false;
```

Comment:
Shouldn't this be removed now?

Reply:
Yes, this is leftover.  I'll remove this.